### ISSUE TITLE
naming of "Exact?" column

### ISSUE THREAD

**Issue by nicholasjhorton** [lindeloev/tests-as-linear#19]
I find the "Exact?" column of the "Common statistical tests are linear models" pdf to be somewhat misleading, since the "Exact?" column links to simulations that show correspondence for sufficiently large n.  My concerns would be alleviated if the column were renamed "Correspondence" or "Equivalence".

I really appreciate this project: nice work!

**Comment by lindeloev** [comment]
Good point. I needed a short word for the cramped sheet. Maybe "Precise" or "Approx." or "Good" or...?

**Comment by nicholasjhorton** [comment]
How about "Equiv." short for Equivalent?

definition: equal in value, amount, function, meaning
